spec/install_spec.rb

```diff
@@ -13,19 +11,12 @@
             stub_chef_zero(platform, version, server) # stub other nodes in chef-zero
           end.converge(described_recipe)
         end
-
-        # any platform specific data you want available to your test can be loaded here
-        _property = load_platform_properties(platform: platform, platform_version: version)
-
-        it 'installs elasticsearch' do
```

Comment:
What's the reason for removing these tests for confirming all the install methods result in a working install?

Reply:
We should just run Test Kitchen here instead. This method, as is. Doesn't work. We'd need to do something like apache2 [does here](https://github.com/sous-chefs/apache2/blob/main/spec/resources/conf_spec.rb#L9)

Reply:
We can definitely move to test kitchen 👍 It was just a lot of test cases originally and faster to catch basic problems like uid/gid differences quickly in a unit test.